README.md

```diff
@@ -1,17 +1,242 @@
 # Review Service
 
-This repository contains a simple contract to prompt user to review the current application on Android and iOS.
+This repository contains simple contracts to prompt the user to review the current application if conditions are met based on provided data for Android and iOS.
 
-[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](LICENSE)
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](LICENSE) ![Version](https://img.shields.io/nuget/v/ReviewService?style=flat-square) ![Downloads](https://img.shields.io/nuget/dt/ReviewService?style=flat-square)
+
+## Cornerstones
+
+- **Customizable**
+  - Every interface, implementation, and extension are using the generic pattern to easily allow more specific review conditions to be used for any application.
+
+- **Declarative**
+  - We are using a builder pattern for the review conditions to easily understand the behavior of the service by looking at its declaration.
+
+- **Testable**
+  - Our interfaces are platform-agnostic, so you can easily test them or test code that uses them.
+
+## Before Getting Started
+
+Before getting started, please read the [Android](https://developer.android.com/guide/playcore/in-app-review) and [iOS](https://developer.apple.com/design/human-interface-guidelines/ratings-and-reviews) application review documentation.
 
 ## Getting Started
 
-{Instructions to quickly get started using the project: pre-requisites, packages
-to install, sample code, etc.}
+1. Add the `ReviewService` NuGet package to your platform projects (Windows, Android and iOS).
+
+2. Add the `ReviewService.Abstractions` NuGet package where it's needed in your project layers (Client, Business and Presentation).
+
+3. Create an implementation for `IReviewSettingsSource` (which holds the review prompt settings used for prompt conditions).
+
+4. Create an instance of `ReviewService`. We recommend to use dependency injection for that and also injecting all required services from the example.
+   ``` cs
+   using ReviewService;
+   using ReviewService.Abstractions;
+
+   var reviewService = new ReviewService<ReviewSettings>(
+       logger: null,
+       reviewPrompter: new ReviewPrompter(logger: null),
+       reviewSettingsSource: new ReviewSettingsSource(),
+       reviewConditionsBuilder: ReviewConditionsBuilder.Empty().MinimumPrimaryActionsCompleted(1)
+   )
+   ```
+
+7. Track and update review settings that are used by review conditions to know if a review should be prompted (let's assume the service was injected and we are using it inside a class).
```

Comment:
I think the last two steps could be combine into "Use the service". Then split into 2 sub points:
- Update the ReviewSettings based on application events (using the track methods).
- Try showing the review prompt when appropriate (using the TryRequestReview method).